torch_xla/core/xla_env_vars.py

```diff
@@ -26,3 +26,4 @@
 TPU_PROCESS_ADDRESSES = 'TPU_PROCESS_ADDRESSES'
 TPU_VISIBLE_DEVICES = 'TPU_VISIBLE_DEVICES'
 TPU_PROCESS_PORT = 'TPU_PROCESS_PORT'
+CPU_ASYNC_CLIENT = 'CPU_ASYNC_CLIENT'
```

Comment:
nit: can you make this `PJRT_CPU_ASYNC_CLIENT` to emphasize this option is only implemented for PJRT?